components/vocabularies/hpo/annotation-phenotypes/src/main/java/org/phenotips/vocabulary/annotation/internal/DiseaseNotSymptomAnnotationExtension.java

```diff
@@ -50,10 +55,12 @@
 
     private static final String NOT_SYMPTOM = "not_symptom";
 
+    private static final Collection<String> TARGET_VOCABULARIES = createTargetVocabularyLabels();
```

Comment:
can we shorten this to  `private static final Collection<String> TARGET_VOCABULARIES = Collections.singletonList(ORPHANET_LABEL);` ?